coffee-module/coffee-module-redis/pom.xml

```diff
@@ -1,23 +1,52 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>hu.icellmobilsoft.coffee</groupId>
-		<artifactId>coffee-module</artifactId>
-		<version>1.11.0-SNAPSHOT</version>
-	</parent>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>hu.icellmobilsoft.coffee</groupId>
+        <artifactId>coffee-module</artifactId>
+        <version>1.11.0-SNAPSHOT</version>
+    </parent>
 
-	<artifactId>coffee-module-redis</artifactId>
+    <artifactId>coffee-module-redis</artifactId>
 
-	<dependencies>
-		<dependency>
-			<groupId>hu.icellmobilsoft.coffee</groupId>
-			<artifactId>coffee-tool</artifactId>
-		</dependency>
+    <dependencies>
+        <dependency>
+            <groupId>hu.icellmobilsoft.coffee</groupId>
+            <artifactId>coffee-tool</artifactId>
+        </dependency>
 
-		<dependency>
-			<groupId>redis.clients</groupId>
-			<artifactId>jedis</artifactId>
-			<version>4.2.3</version>
-		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>redis.clients</groupId>
+            <artifactId>jedis</artifactId>
+            <version>4.2.3</version>
+        </dependency>
+
+        <!--Test dependecies -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.weld</groupId>
+            <artifactId>weld-junit5</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.smallrye.config</groupId>
+            <artifactId>smallrye-config</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.smallrye.config</groupId>
+            <artifactId>smallrye-config-source-yaml</artifactId>
+            <version>2.8.1</version>
```

Comment:
szintúgy verziós kérdés, hol legyen?

Reply:
ez is mivel test scope es lokalis modulban van csak a teszteknek, igy jobb lehet. Vagy itt is bevezethetunk olyan bom/pom.xml-t ami a tesztekkel foglalkozik. De az nem ez a issue jelenleg.